h/search/core.py

```diff
@@ -145,15 +178,10 @@ def _instrument(self):
             s.send()
 
     @staticmethod
-    def _default_querybuilder(request):
+    def _default_querybuilder(request, filters, request_filters):
```

Comment:
We could split this into two separate methods; one that adds reg filters and one that adds request filters. Not sure if that's necessary or not??

Reply:
This looks fine to me.